Sources/Slacket/Slacket.swift

```diff
@@ -37,5 +37,6 @@ struct Slacket: ServerModuleType {
         router.get("/", middleware: StaticFileServer(path: repoDirectory+"public/"))
         router.all("api/*", middleware: BodyParser())
         router.all("api/*", middleware: SlacketHandler())
+        router.get("/health-check", middleware: HealthCheckMiddleware())
```

Comment:
Routes are processed in the order they are added, so for every request that will be the last one to check. Since health check is performed very often we would want to return from it as soon as possible. Could you please move this route to be the first added?